kuma/javascript/src/payments/subheaders/signup.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as React from 'react';
+import { gettext, interpolate } from '../../l10n.js';
+import SubHeader from './index.jsx';
+import SubscriptionForm from '../subscription-form.jsx';
+
+type Props = {
+    num: ?number,
+    showSubscriptionForm: ?boolean,
+};
+
+export const title = gettext('Become a monthly supporter');
+export const subtitle = gettext('You will be MDN member number: %(num)s');
+export const description = gettext(
+    'Support MDN with a %(amount)s monthly subscription and get back more of the knowledge and tools you rely on for when your work has to work.'
+);
+
+const SignupSubheader = ({ num, showSubscriptionForm }: Props): React.Node => {
+    const subtitleInterpolate = interpolate(gettext(subtitle), {
+        num: num && num.toLocaleString(),
+    });
+    const descriptionInterpolate = interpolate(gettext(description), {
+        // Hard-coded until this issue is addressed: https://github.com/mdn/kuma/issues/6654
+        amount: '$5',
+    });
+
+    return (
+        <SubHeader
+            title={title}
+            subtitle={subtitleInterpolate}
+            description={descriptionInterpolate}
+            classNames={showSubscriptionForm ? 'has-form' : ''}
```

Comment:
Is there really no way to avoid this? 
It's just a bit odd that the form has to be a prop rather than `children`. It would have made so much more sense if it could have been...
```jsx
<SignupSubheader
                    num={subscriberNumber}
>
  {showSubscriptionForm && <SubscriptionForm/>}
</SignupSubheader>
```

At least, if we can't live without the information about `has-form` can we at least refactor it to:

```jsx
<SignupSubheader
                    num={subscriberNumber}
                   showSubscriptionForm={showSubscriptionForm}
>
  {showSubscriptionForm && <SubscriptionForm/>}
</SignupSubheader>
```
...and the necessary refactoring inside `SignupSubheader` now that it no longer needs to actually know about the form.

Reply:
I removed the classname prop, but I strongly think `<SubscriptionForm />` makes the most sense to live in `<SignupSubheader />`. It feels too much like prop drilling to pass it as a child from the Payments Landing page and then again as a child from `<SignupSubheader />`. 

I think another convenience is that if I wanted this sub header on another page for whatever reason, I can import `<SignupSubheader />` and trust that it handles rendering the form, rather than having to import `<SubscriptionForm />` separately and pass it in as a child. There's nothing in the designs, as of now, that indicates that we might want that right-hand content in the sub header to be more dynamic. If that changes, then I think passing a child to `<SignupSubheader />` would be an option.

Reply:
As you feel. It's your call.